MatrixInfoApp/config.yaml

```diff
@@ -0,0 +1,56 @@
+# do not touch this line
+configVersion: 2
+
+name: MatrixInfoApp
+description: This app shows informations about all sensor on app dashboard
+keywords: 'sensor, info, dash'
+
+screens:
+  - - sensorDetails
+  - - gyroscopeDetails
+  - - uvDetails
+
+sensors:
+  - temperature
+  - humidity
+  - gyroscope
+  - uv
+  - pressure
+  - altitude
+
+dataTypes:
+  sensorValue:
+    Type Sensor: string
```

Comment:
`Invalid key:  Type Sensor  Data types must only be named using letters, numbers and "_".`